client/src/index.tsx

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
Did this get added to all these files because of the new linting?

Reply:
Yes